CHANGES.rst

```diff
@@ -23,6 +23,10 @@ Compute
   (GITHUB-1661, GITHUB-1661)
   [Dimitris Galanis - @dimgal1]
 
+- [OpenStack] Fix error creating and getting node in OpenStack.
```

Comment:
Please add some more details aka that it only affected scenarios where ``ex_force_microversion`` was set to ``> 2.4.7``.

Reply:
Updated.